doc/api/diagnostics_channel.md

```diff
@@ -307,6 +307,79 @@ channel.subscribe(onMessage);
 channel.unsubscribe(onMessage);
 ```
 
+### Built-in Channels
+
+* `http.client.request.start`: Emited with arguments `request` when client
+  starts a request.
+* `http.client.response.finish`: Emited with arguments `request`, `response`
+  when client receives a response.
+* `http.server.request.start`: Emited with arguments `request`, `response`,
+  `socket`, `server` when server receives a request.
+* `http.server.response.finish`: Emited with arguments `request`, `response`,
+  `socket`, `server` when server sends a response.
```

Comment:
Perhaps structure this more like the argument type lists with each event having sub-bullets for each value in the object showing the name, type, and a description?

Reply:
> 

Thanks ! Have updated, please help review again 😊 !

Reply:
> Perhaps structure this more like the argument type lists with each event having sub-bullets for each value in the object showing the name, type, and a description?

Thanks ! Have updated, please help review again 😊 !